tests/aws-cpp-sdk-dynamodb-integration-tests/TableOperationTest.cpp

```diff
@@ -77,8 +77,8 @@ Aws::String BuildTableName(const char* baseName)
 class TableOperationTest : public ::testing::Test {
 
 public:
```

Comment:
These changes allows tests to be run individually using gtest_filter enabling developers to debug case by case, as opposed to having to run all the tests for just even one test to run successfully.

Change is moving from static setup/cleanup to fixture level setup/cleanup in line with how S3 tests have it